pkg/apis/core/validation/validation.go

```diff
@@ -3850,6 +3859,13 @@ func ValidatePodStatusUpdate(newPod, oldPod *core.Pod) field.ErrorList {
 	allErrs = append(allErrs, ValidateContainerStateTransition(newPod.Status.ContainerStatuses, oldPod.Status.ContainerStatuses, fldPath.Child("containerStatuses"), oldPod.Spec.RestartPolicy)...)
 	allErrs = append(allErrs, ValidateContainerStateTransition(newPod.Status.InitContainerStatuses, oldPod.Status.InitContainerStatuses, fldPath.Child("initContainerStatuses"), oldPod.Spec.RestartPolicy)...)
 
+	if newIPErrs := validatePodIPs(newPod); len(newIPErrs) > 0 {
+		// Tolerate IP errors if IP errors already existed in the old pod. See http://issue.k8s.io/90625
```

Comment:
Add a TODO stating when this toleration can be removed?

Reply:
hmm, it'll read like "TODO: stop tolerating this once we can guarantee the existing pod was not created prior to 1.19"... I'm not aware of a plan to accomplish that

Reply:
Kubelet maintains these, I think? Invalid pods shouldn't persist more than a few seconds. I think you just need to tolerate for one release?

Reply:
added a TODO for 1.20